pkg/changelog/changelog.go

```diff
@@ -0,0 +1,254 @@
+package changelog
```

Comment:
Can we put that under `pkg/release` to indicate this is releases' changlog?

Reply:
Done.